front/app/src/js/components/pages/reset_password/ResetPasswordEmail.js

```diff
@@ -74,10 +82,43 @@ export class ResetPasswordEmail extends Component {
   }
 
   async #sendEmail() {
-    const newComponent = document.createElement(
-        'reset-password-code-component',
-    );
+    this.#startLoadButton();
+    try {
+      const {response, body} = await userManagementClient.sendResetPasswordCode(
+          this.email.value,
+      );
+      if (response.ok) {
+        this.#loadCodeComponent();
+      } else {
+        this.#resetLoadButton();
+        this.alertForm.setAttribute('alert-message', body.errors[0]);
+        this.alertForm.setAttribute('alert-display', 'true');
+      }
+    } catch (error) {
+      ErrorPage.loadNetworkError();
+    }
+  }
+
+  #loadCodeComponent() {
+    // const newComponent = document.createElement(
+    //     'reset-password-code-component',
+    // );
```

Comment:
Is this comment intentional ?

Reply:
No, I forgot to delete it thanks!

Reply:
https://github.com/tdameros/42-transcendence/pull/223/commits/df838a1863c469eaaab4890f0725960467c42ac1